plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAlluxioCacheFileOperations.java

```diff
@@ -145,6 +145,11 @@ public void testCacheCheckpointFileOperations()
         assertFileSystemAccesses(
                 "SELECT * FROM test_checkpoint_file_operations",
                 ImmutableMultiset.<CacheOperation>builder()
+                        // Checkpoint read positions and lengths are unstable so we set them to zero
```

Comment:
Could we update the test to have `_change_data`, deletion vector and json delta log entry files as well ?

Reply:
ya, i think it'll be better to register a table on fixed data if that's what it takes to cover more cases and make the test stable.

Reply:
Also, ANALYZE the test table to make sure we're covering the stats json files in the test as well